webpack.appRendererServer.js

```diff
@@ -2,7 +2,7 @@ const path = require('path');
 
 module.exports = {
   target: 'node',
-  entry: ['@babel/polyfill', './src/helpers/renderAppRServerBundle.js'],
+  entry: ['./src/helpers/renderAppRServerBundle.js'],
   output: {
     filename: 'appRendererServerBundle.js',
     path: path.resolve(__dirname, './build'),
```

Comment:
Since we changed it so the server bundle doesn't include the Express server code, are the Webpack configs for the server and App-Renderer server the same now?

Reply:
Do you think we should combine the two webpacks and do something like this with parameters?

module.exports = isAppRendererServerBundle => {
    return {
        entry: isAppRendererServerBundle ? "./renderAppRServerBundle": "./renderApp",
        output: {
          filename: isAppRendererServerBundle ? "appRendererServerBundle.js": "./server.js",
        },
    }
};

or do you think we should just keep them as separate webpack configs?

Reply:
Ah, I see, 2 entrypoints.

You don't have to have a ternary , you can just make an object in `entry` and define a `server` and `app-renderer-server` key for both.

Either way, what are the differences between the Webpack configs? Just the Webpack-Node-Externals plugin?

Reply:
hmm something like ? 

```
module.exports = serverType => {
    return {
        entry: {
           server: './renderApp',
           app-renderer-server: './renderAppRServerBundle',
        }
        output: {
          filename: {
              server: 'server.js',
              app-renderer-server: 'appRendererServerBundle.js',
          }
        },
    }
};
```

The difference is that `./renderApp` returns 

```
export default (req,res,resolve) => {
  const content = renderToString(
    <App/>
  );
  
  return `
    <div id="root">${content}</div>
    <script src="/client.js"></script>
  `
};
```

while `appRendererServerBundle.js` returns 

```
export default (req,res,resolve) => {
  const content = renderToString(
    <App/>
  );
  

  // required app-r template to work
  resolve({
      status: 200,
      markup: `
        <div id="root">${content}</div>
        <script src="http://localhost:3001/client.js"></script>
        `
    })
};
```

the functions are just fundamentally different